lib/stat_tracker.rb

```diff
@@ -10,6 +10,7 @@ def initialize(location)
     @games = {}
     @teams = {}
     @game_teams = {}
+    @min_team = 0
```

Comment:
I see you added this class variable, but I don't see where else it is used in your methods. Do we actually need this?

Reply:
Oh I didn't want that anymore.  Not intentional.  I just updated it now.